cmd/agent/app/options/options.go

```diff
@@ -184,7 +184,8 @@ func (o *Options) AddFlags(fs *pflag.FlagSet, allControllers []string) {
 		"The duration the clients should wait between attempting acquisition and renewal "+
 		"of a leadership. This is only applicable if leader election is enabled.")
 	fs.StringVar(&o.KarmadaKubeConfig, "karmada-kubeconfig", o.KarmadaKubeConfig, "Path to karmada control plane kubeconfig file.")
-	fs.StringVar(&o.KarmadaContext, "karmada-context", "", "Name of the cluster context in karmada control plane kubeconfig file.")
```

Comment:
I guess the `--karmada-context` is exactly the context of the Karmada control plane, not the host cluster.
So, why rename it?

Reply:
oh I though host and karmada would be the same
if they are not then I need to change https://github.com/karmada-io/karmada/pull/5016 to be karmada-as/karmada-group/karmada-uid

Reply:
Yeah, I apologize that I misunderstood your use case, I thought you wanted to impersonate users on host clusters, but after revisiting the original issue (#4695), I figured out the user that you need to impersonate is on the Karmada control plane.

> if they are not then I need to change https://github.com/karmada-io/karmada/pull/5016 to be karmada-as/karmada-group/karmada-uid

+1 Yes, that's the one we talked about at https://github.com/karmada-io/karmada/issues/4695#issuecomment-1989757658.

Reply:
By the way, we usually call a cluster `host` because the cluster will host Karmada's components.
So, the `--host` might only be present in `karmadactl init` or `karmadactl addon` command, which is used to install components on a host cluster.

Reply:
kk, so I'll send a rename PR for the -as stuff :)

Reply:
@RainbowMango https://github.com/karmada-io/karmada/pull/5099